app/controllers/admin/user_creations_controller.rb

```diff
@@ -31,17 +31,6 @@ def hide
     elsif @creation_class == ExternalWork || @creation_class == Bookmark
       redirect_to(request.env["HTTP_REFERER"] || root_path)
     else
-      unless action == "unhide" || @creation_class == Work
```

Comment:
Although the comment above the method says "creations," `UserMailer.admin_hidden_work_notification` is _only_ for works: https://github.com/otwcode/otwarchive/blob/7570b001476c05306fe9b47172fdfd6432dc02ee/app/mailers/user_mailer.rb#L359 

Since sending that email is handled here: https://github.com/otwcode/otwarchive/blob/58c1dd964fc6b7a2b6ae19adabc9ea59a00da965/app/models/work.rb#L1454

There's no reason to leave this hanging around, unless we enjoy errors when hiding anything that isn't a Work, Bookmark, or External Work.

Reply:
Is it worth it to leave parts of this code here (maybe commented out, or in an unreachable block) to make it easier for someone to eventually fix [AO3-4314](https://otwarchive.atlassian.net/browse/AO3-4314)? Or is it better to keep the code clean and have whoever ends up tackling that bug rewrite it from scratch?

Reply:
Grumble, grumble, apparently replying by email doesn't put replies in threads. Deleting and pasting here:

I'm more inclined to let them start from scratch -- better logic, different mailer -- assuming we decide to add an email. (Abuse doesn't generally need to hide series, hence not even realizing we'd broken this some time ago. And when they do hide or delete things, they always contact users directly and don't just rely on the automated email.)